card/mod/02_basic_types/set/type/pointer.rb

```diff
@@ -4,6 +4,11 @@
   self.drop_item Env.params['drop_item'] if Env.params['drop_item']
 end
 
+event :insert_item_event, :before=>:approve, :on=>:save, :when=> proc {|c| Env.params['insert_item']} do
+  index = Env.params['item_index'] || 0
+  self.insert_item index.to_i, Env.params['insert_item']
+end
+
```

Comment:
This is the only siginificant change in this PR. I need this for wikirate's profile pages to change position of pointer items. Maybe arguable whether we want to support this in wagn.